upp-rds-provisioner/README.md

```diff
@@ -50,8 +54,28 @@ psql -h localhost -p 5432 -U <DB_USERNAME> Factset
 
 An alternative to using `psql` is to use [pgAdmin](https://www.pgadmin.org/).
 
-Todo
-------
-1. Move this into repo for all our infrastructure
-2. Add cloudwatch alarms
-3. Implement lower environment teardown over weekends and recreate on mondays
+
+## Decommisioning a cluster
+- Export the required environment variables.
+- Run the following Docker command:
+```
+docker pull coco/upp-rds-provisioner:latest
+docker run \
+    -e "CLUSTER=$CLUSTER" \
+    -e "AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION" \
+    -e "ENVIRONMENT_TYPE=$ENVIRONMENT_TYPE" \
+    -e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
+    -e "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" \
+    -e "VAULT_PASS=$VAULT_PASS" \
+    -e "KONSTRUCTOR_API_KEY=$KONSTRUCTOR_API_KEY" \
+    coco/upp-rds-provisioner:latest decom.sh
+```
+- `Delete RDS cluster` step may take up to 5 minutes, as CloudFormation waits until the RDS instance is fully decommissioned before returning a success code.
+- You can monitor the provisioning by going to the Cloudformation section in the AWS console and looking for the stack `upp-<CLUSTER>-factset-rds`.
+- If you've got the permissions, you can also delete the cloudformation stack directly from the AWS console. 
```

Comment:
Most people won't have the correct permissions IIRC - let's just get people to use the Docker image, as it means we know that it's always going to be tested and consistent, and nothing will get missed that way :)